clients/csharp-uwp/README.md

```diff
@@ -19,7 +19,7 @@ Specifically, the sample demonstrates how to
 
 ## Prerequisites
 
-- A Limited Access Feature token issued by Microsoft.
+- [optional] A Limited Access Feature token issued by Microsoft.
```

Comment:
Agreed that this is confusing. Can we add an additional note about why it's optional or what you'd need it for, like the point below? Strawman draft:

`[optional] A Limited Access Feature token, issued by Microsoft, needed if you are building, packaging, or publishing your own app. For development purposes, the sample already contains an appropriate token.`